### ISSUE TITLE
🐛[BUG][Combat] Ouryu Cometh Issues

### ISSUE THREAD

**Issue by siknoz** [HorizonFFXI/HorizonXI-Issues#1512]
1) When Ouryu is on the ground he never used Typhoon Wing
2) Geotic Breath doesn't trigger if you stand on the feet
3) If you keep inching closer towards the center until he's able to use Geotic Breath it does 1400+ damage
4) Absolute Terror is single target

**Steps to Reproduce**
1) Use Mistmelts whenever he flies, give tp and won't see any Typhoon Wings. It's entirely possible that we somehow got extremely lucky and he just never used it based on random, but doing 3 full fights without 1 wing is suspicious.
2) Stand on the outside feet, force Ouryu to use Geotic Breath, watch him not use it.
3) Same as  step 3 above, just keep inching towards the center .1 at a time and eventually he'll use geotic breath and hit, hits for 1400+
4) Have Ouryu use Absolute Terror

**Expected behavior**
1) Should use Typhoon Wing on the ground 
2) Should always hit at least its current target with breath when used
3) None of the in-era videos I could find show geotic breath damage, however this allakazham post mentions that a tank got hit for 780 which is at least something. https://ffxi.allakhazam.com/db/bestiary.html?fmob=3484#m11448006432791
4) Absolute Terror should be AOE - https://ffxiclopedia.fandom.com/wiki/Absolute_Terror

**Comment by TracentEden** [comment]
@siknoz Yep you are correct there is issue where it can only currently use Typhoon Wing on the first ground phase before it's first flight, will fix.

**Comment by TracentEden** [comment]
Points 1 and 3 have been fixed, adding point 4 in an upcoming PR. @siknoz Do you have source that shows such wyrm breath attack should still happen even if player if not in front of the mob? The current code for all such breaths prevents these moves if the players are at a certain angle away from the front, do you think this angle is wrong or that there should be no such limiting condition?

**Comment by TracentEden** [comment]
Or do you mean the condition where it was trying to use it but could not because of the mob finding target system?
